src/components/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -27,7 +36,8 @@ export interface VideoThumbnailProps {
 
 export function VideoThumbnail({
   thumbnailUrl,
-  videoLength,
+  videoLength = 0,
+  videoProgress = 0,
```

Comment:
How do you feel about making the API more explicit and having a separate prop for showing the video progress (such as `showVideoProgress`), rather than relying on `progressValue` for this logic?

Reply:
Having an additional prop won't change the fact that the video progress is dependent on video length also being set. We should be clear in the prop description that it will only render if `videoLength` is set 💯.

Reply:
That's a good point!

I also thought of one case where having a `showVideoProgress` prop could be useful: if for some reason we want to show the progress bar when `videoProgress=0` (default).

Reply:
Great suggestions!

If we want to add `showVideoProgress`, should we should also add `showVideoLength` to make component's API consistent? However, I feel it might be a bit overkill to maintain two additional props on component (at least based on the use cases we have for VideoThumbnail on web).

I would be totally down to add both props if you guys think they should be added? 🙂

Reply:
That's a good question! If there is a use-case for the video length to be hidden from the user, then `showVideoLength` would make sense as a prop. If that's resulting in inconsistencies and poorer experiences, I'm not sure this is something we'd want.

Reply:
I do not think there is a use-case for hiding video length through separate prop. Video progress does not make a lot of sense without video length displayed.

But to address original question, @kaelig do you feel strong on adding `showVideoProgress` as a prop?

Reply:
Gotcha, thanks!

I'm a 7 out of 10 on having a clear `showVideoProgress` in there rather than having logic in the component that "smartly" determines if the video progress should appear. That way, we get a fully declarative API rather than a data-based API (which is typically easier to learn and debug).

Reply:
Ok great, thanks. Will add new prop asap!